config/externalname.go

```diff
@@ -376,6 +376,10 @@ var terraformPluginSDKExternalNameConfigs = map[string]config.ExternalName{
 	"google_data_catalog_tag": config.IdentifierFromProvider,
 	// {{name}}: projects/{project_id}/locations/{location}/tagTemplates/{tagTemplateId}
 	"google_data_catalog_tag_template": config.TemplatedStringAsIdentifier("tag_template_id", "projects/{{ .setup.configuration.project }}/locations/{{ .parameters.region }}/tagTemplates/{{ .external_name }}"),
+	// {{name}} projects/{project}/locations/{region}/taxonomies/{taxonomy}
+	"google_data_catalog_taxonomy": config.IdentifierFromProvider,
+	// {{name}} projects/{project}/locations/{region}/taxonomies/{taxonomy}/policyTags/{policytag}
+	"google_data_catalog_policy_tag": config.IdentifierFromProvider,
```

Comment:
Note: Since `taxonomy` and `policytag` are system generated identifiers, we cannot use `TemplatedStringAsIdentifier` here: `id: projects/official-provider-testing/locations/eu/taxonomies/2064837193133375339/policyTags/1548338660528116821`